### ISSUE TITLE
 Background and foreground colors do not have a sufficient contrast ratio

### ISSUE THREAD

**Issue by jcabak** [inclusive-design/wecount.inclusivedesign.ca#1739]
## Describe the issue

Color for "An [Inclusive Design Research Centre](https://idrc.ocadu.ca/) project" in the top bar do not have a sufficient contrast ratio.
Currently contrast ratio is 3.37

## To Reproduce

N/A

## Expected behavior

Contrast ratio should be at least 4.5:1

## Screenshots

N/A

## Applicable WCAG Success Criteria:

- [1.4.3 - Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)

## Desktop:

N/A

## Mobile device:

N/A

## Additional context

N/A

**Comment by carenwatkins** [comment]
suggest changing to black and underline 
<img width="414" alt="Screenshot 2024-03-11 at 1 04 26 PM" src="https://github.com/inclusive-design/wecount.inclusivedesign.ca/assets/11746584/d06180e0-51f2-4412-ba50-7ec99b91051b">

**Comment by greatislander** [comment]
Thanks for catching this, @jcabak — I think I'm going to close your PR and go with my colleague's design suggestion instead, which will resolve the issue you found.
